doc/Customization-guide/How-to-contribute/Documentation-contribution-and-style-guide/index.md

```diff
@@ -230,7 +337,7 @@ Key differences between contributing source code and documentation are:
   * [Jekyll-paginate](https://github.com/jekyll/jekyll-paginate)
   * [Rouge](https://github.com/jneen/rouge)
 
-   Detailed installation instructions for selected platforms:
+   See the detailed installation instructions in the table below. Click the appropriate tab to select your platform:
```

Comment:
Ok, I understand. But I don't think we should mention that :) My edit was towards making it a call-to-action description. I'll rephrase the 2nd sentence this way: 'Click the tab to select a platform'.